diff --git a/main/Kconfig.projbuild b/main/Kconfig.projbuild
--- a/main/Kconfig.projbuild
+++ b/main/Kconfig.projbuild
@@ -235,4 +235,7 @@ choice BOARD_TYPE
     config BOARD_TYPE_ESP_VOCAT
         bool "Espressif ESP-VoCat"
         depends on IDF_TARGET_ESP32S3
+    config BOARD_TYPE_MEMORIA_ESP_VOCAT
+        bool "Memoria ESP-VoCat N32R16"
+        depends on IDF_TARGET_ESP32S3
     config BOARD_TYPE_ESP_SPARKBOT
diff --git a/main/CMakeLists.txt b/main/CMakeLists.txt
--- a/main/CMakeLists.txt
+++ b/main/CMakeLists.txt
@@ -305,4 +305,9 @@ elseif(CONFIG_BOARD_TYPE_ESP_VOCAT)
     set(EMOTE_RESOLUTION "360_360")
     set(ESP_VOCAT_ASSETS_STAGING "${CMAKE_BINARY_DIR}/esp-vocat-assets")
     set(EMOTE_EXTERNAL_PATH "${ESP_VOCAT_ASSETS_STAGING}")
+elseif(CONFIG_BOARD_TYPE_MEMORIA_ESP_VOCAT)
+    set(BOARD_DIR "memoria/esp-vocat")
+    set(BUILTIN_TEXT_FONT font_noto_sans_basic_20_4)
+    set(BUILTIN_ICON_FONT font_material_symbols_20_4)
+    set(DEFAULT_EMOJI_COLLECTION noto-color-emoji_64)
 elseif(CONFIG_BOARD_TYPE_ESP_SENSAIRSHUTTLE)
