diff --git a/main/application.cc b/main/application.cc
index 2d464e4..0000000 100644
--- a/main/application.cc
+++ b/main/application.cc
@@ -62,4 +62,7 @@ void Application::Initialize() {
     // Setup the display
     auto display = board.GetDisplay();
     display->SetupUI();
+#if CONFIG_BOARD_TYPE_MEMORIA_ATK_DNESP32S3_V1
+    display->ShowNotification("Memoria", 1500);
+#endif
     // Print board name/version info
     display->SetChatMessage("system", SystemInfo::GetUserAgent().c_str());
